utils/registerParams.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
```

Comment:
Whothis? What's this script for?

Reply:
It loads config data into AWS Parameter Store. README has some info on that. I was thinking it might be neater to use a JSON dist file because I want to add more configuration options over time. But for now, this is a spiffy menu interface.